geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateDefinedIndexesCommand.java

```diff
@@ -143,10 +152,19 @@ public Result createDefinedIndexes(
       result = ResultBuilder.createGemFireErrorResult(e.getMessage());
     }
 
-    if (xmlEntity.get() != null) {
-      persistClusterConfiguration(result,
-          () -> getSharedConfiguration().addXmlEntity(xmlEntity.get(), group));
+    if (!xmlEntities.isEmpty()) {
```

Comment:
this is unnecessary. If it's empty, the loop will have 0 iteration.

Reply:
👍